protected/views/site/guide.php

```diff
@@ -23,75 +12,86 @@
                     <li><a href="/">Home</a></li>
                     <li class="active">Guidelines</li>
                 </ol>
-                <h4>GigaDB - Submission Guidelines</h4>
+                <h1 class="h4">GigaDB - Submission Guidelines</h1>
             </div>
         </section>
         <section>
 
-            <section style="margin-bottom: 5px;">
-                <div style="display:inline-block;">
-                    <ul class="nav nav-tabs nav-border-tabs" style="margin-top: 1px; margin-bottom: 1px">
-                            <li class="active"><a href="/site/guide">General Submission Guidelines</a></li>
-                            <li class="dropdown">
-                                <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
-                                    Datasets Checklists&nbsp;<i class="fa fa-angle-down"></i>
-                                </a>
-                                <ul class="dropdown-menu" style="margin-top: 5px;">
-                                    <li><a href="/site/guidegenomic">Genomic Dataset Checklist</a></li>
-                                    <li><a href="/site/guideimaging">Imaging Dataset Checklist</a></li>
-                                    <li><a href="/site/guidemetabolomic">Metabolomic and Lipidomic Dataset Checklist</a></li>
-                                    <li><a href="/site/guideepigenomic">Epigenomic Dataset Checklist</a></li>
-                                    <li><a href="/site/guidemetagenomic">Metagenomic Dataset Checklist</a></li>
-                                    <li><a href="/site/guidesoftware">Software Dataset Checklist</a></li>
-                                </ul>
-                            </li>
-                </div>
-            </section>
+        <?php
+            $this->widget('application.components.GuideNavigation');
+        ?>
             <section>
                 <div class="tab-content">
-                    <div role="tabpanel" class="tab-pane active" id="general">
```

Comment:
this and all subsequent `role="tabpanel"` are removed since the context is not a `tablist` and the widget that looks like a tab is more fitting of a secondary `nav` with a `disclosure` button (dropdown menu)